### PR TITLE
Documentation update according to changes in Vehicle Template

### PR DESCRIPTION
**Pull request by mml5bg**
## Describe your changes
Documentation update according to the changes implemented in https://github.com/eclipse-velocitas/vehicle-app-python-template/pull/15
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: content/en/docs/tutorials/vehicle-app-runtime/run_runtime_services_locally.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ aliases:
 
 * Dapr (```Local - Ensure Dapr```): installs Dapr CLI and initializes Dapr if required
 * Mosquitto (```Local - Mosquitto```): runs _Mosquitto_ as a container (```docker run```)
-* KUKSA Data Broker (```Local - VehicleDataBroker```): downloads and runs _KUKSA Data Broker_
-* (Optional) Seat Service (```Local - SeatService```): downloads and runs _Seat Service_, an example `Vehicle Service`
-* (Optional) Feeder Can (```Local - FeederCan```): downloads and runs _FeederCAN_
+* KUKSA Data Broker (```Local - VehicleDataBroker```): runs _KUKSA Data Broker_ as a container
+* (Optional) Vehicle Services (```Local - VehicleServices```): runs _Vehicle Services_, an example `Seat Service` as a container
```
Comment (BjoernAtBosch) [inline content/en/docs/tutorials/vehicle-app-runtime/run_runtime_services_locally.md:19]:
```suggestion
* (Optional) Vehicle Services (```Local - VehicleServices```): runs the _Vehicle Services_ (e.g. the `Seat Service`) configured in the `AppManifest.json` as each a separate container
```

File: content/en/docs/tutorials/vehicle-app-runtime/run_runtime_services_locally.md [hunk 2]
```diff
@@ -15,9 +15,9 @@ aliases:
 
 * Dapr (```Local - Ensure Dapr```): installs Dapr CLI and initializes Dapr if required
```
Comment (BjoernAtBosch) [inline content/en/docs/tutorials/vehicle-app-runtime/run_runtime_services_locally.md:16]: I've a discussion point for a bigger round (if my understanding is correct):

Regarding "Local" vs. "K3D": Maybe I've a limited understanding, but is "local" a good "opposite" to "K3D"? I mean is the run in k3d not also done on the local machine?

Wouldn't it then better to talk about "unmanged" and "K3D"?

**Review by BjoernAtBosch — CHANGES_REQUESTED** [review]

File: content/en/docs/tutorials/vehicle-app-runtime/run_runtime_services_locally.md (hunk 2 above)
Comment (mml5bg) [reply content/en/docs/tutorials/vehicle-app-runtime/run_runtime_services_locally.md:16]: From my point of view it is correct, but let's discuss about it in bigger round, this is not in scope of this PR

Comment (BjoernAtBosch) [reply content/en/docs/tutorials/vehicle-app-runtime/run_runtime_services_locally.md:16]: Yes, Dennis was also asking about that. I asked him to create a task for that.

**Review by BjoernAtBosch — APPROVED** [review]

**Pull request merged**